installing/installing_aws/manually-creating-iam.adoc

```diff
@@ -3,6 +3,8 @@
 include::modules/common-attributes.adoc[]
```

Comment:
These changes have been reviewed in the other PR for 4.7 CCO work, but I copied them over since that one isn't merged yet.